mod-source-record-storage-server/src/main/java/org/folio/services/handlers/actions/ModifyRecordEventHandler.java

```diff
@@ -145,4 +164,12 @@ public boolean isPostProcessingNeeded() {
   public String getPostProcessingInitializationEventType() {
     return DI_SRS_MARC_BIB_RECORD_MODIFIED_READY_FOR_POST_PROCESSING.value();
   }
+
+  private OkapiConnectionParams retrieveOkapiConnectionParams(DataImportEventPayload eventPayload) {
```

Comment:
We can use RestUtil.retrieveOkapiConnectionParams here

Reply:
sure, just turned out this refactoring has not been pushed yet, thanks!